shaping/render_test.go

```diff
@@ -0,0 +1,195 @@
+package shaping
```

Comment:
We might want to look at whether this is needed in the typesetting repo as there is more render work in the `render` repository, it would be a shame to duplicate for test purposes.

Reply:
I agree this file seems a bit out of place, but, in the meantime, I found this quick and dirty visualisation tool very handy when developing. 

As you said, a better solution would be to use `render`. Perhaps we can keep this file for now, and once `render` supports vertical text, move the examples there and remove `shaping/render_test.go` ?